puppet/files/treeherder/.bash_aliases

```diff
@@ -90,3 +91,11 @@ function thresetall {
     echo "Purging queues"
     thqueuespurge
 }
+
+function thpurge {
+    ~/treeherder/manage.py init_datasources --reset
+    ~/treeherder/manage.py flush
+    thresetall
+    echo
+    echo "Please exit vagrant and run 'vagrant provision' to finish"
```

Comment:
Could you land this as a quick followup on master at some point? (no review needed) :-)